gapic/batch/common.yaml

```diff
@@ -0,0 +1,5 @@
+common:
+  batch_apis: '*'
+  batch_languages: '*'
+  api_config_pattern: ${THISDIR}/../api/artman_*.yaml
+  artman_language_yaml: ${THISDIR}/../lang/common.yaml
```

Comment:
I can't remember the full justification now. In any case, it seems simpler to have everything consistent. It's up to you if you want to change gapic/lang/common.yaml, but I at least wanted to apply the rule to new files.